example_project/urls.py

```diff
@@ -6,4 +6,5 @@
 urlpatterns = [
     path("hello/", views.hello_world, name="hello-world"),
     path("my_account/", views.my_account, name="my-account"),
+    path("/login-failure", views.login_failure, name="login-failure"),
```

Comment:
Nitpicking point, could make it `login-failure/` to be consistent with other URLs? (not essential)

Reply:
[caf1eff](https://github.com/i-dot-ai/automatilib/pull/24/commits/caf1effa8c8985aa6c04fcf142bbf051934f3d37)